database/DatabaseExampleSwiftUI/DatabaseExample/Shared/Models/Post.swift

```diff
@@ -24,6 +24,7 @@ struct Post: Identifiable {
   var title: String
   var body: String
   var starCount: Int
+  var starDictionary: [String: Bool]
```

Comment:
I wonder if `Post` model is used anywhere now? In a real app we would have more rich Model layer where we would encapsulate Firebase logic. For demo purposes using RTDB directly from View Models looks fine to me unless we have to duplicate the logic between View Models. So it looks like `Post` model can be removed so far.

@paulb777 @ryanwilson @ncooke3 Any thoughts on it?